### PR TITLE
Revert "Revert "Bump Pagerfanta from 2.x to 3.x""

### PR DESCRIPTION
**Pull request by lchrusciel**
| Q               | A
| --------------- | -----
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | reverts #416 for higher branches
| License         | MIT

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -58,6 +58,7 @@
         "doctrine/orm": "^2.5",
         "lchrusciel/api-test-case": "^5.0",
         "matthiasnoback/symfony-dependency-injection-test": "^4.2.1",
+        "pagerfanta/pagerfanta": "^3.0",
```
Comment (loic425) [inline composer.json:61]: for now I suggest you to move it on main requirements. We'll handle the "optional" requirements later on next release

**Review by loic425 — CHANGES_REQUESTED** [review]

**Review by GSadee — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (lchrusciel) [reply composer.json:61]: it was just a revert to make our branches green again, so I would go with it for now. But I agree, that 1.9 needs this. The question is should be part of this PR?

Comment (loic425) [reply composer.json:61]: Hum good idea to make this change in another "Fix" PR.

**Review by loic425 — APPROVED** [review]

**Comment by lchrusciel** [comment]
Thank you, Łukasz! :tada:

**Pull request merged**